### PR TITLE
Update audience operation support details in Velocity

### PR DESCRIPTION
**Pull request by Timongcraft**
- Documented support for tablist headers, footers, and resource packs.
- Clarified that unsupported operations include sounds and books. (currently the only unsupported operations)

Support for resource packs was added to the `ConnectedPlayer` class [here](https://github.com/PaperMC/Velocity/commit/cbd07b14349c449202e84a840da1167b42487cca#diff-1cd097c1de92fff8c43bc33eb4f52405eb5396f9e8cc922a4502e4cad12e1f88).
Support for the tablist header and footer was added [here](https://github.com/PaperMC/Velocity/commit/5da085d82f9cd7a91a4b8d44f6dc94dc3383b173#diff-1cd097c1de92fff8c43bc33eb4f52405eb5396f9e8cc922a4502e4cad12e1f88).

Link to the original page: https://docs.papermc.io/velocity/dev/pitfalls#audience-operations-not-supported
Link to the page: https://timongcraft-fix-audience-api.papermc-docs.pages.dev/velocity/dev/pitfalls#audience-operations-are-not-fully-supported

### CODE REVIEW

File: docs/velocity/dev/getting-started/pitfalls.mdx [hunk 1]
```diff
@@ -33,5 +33,5 @@ public void onProxyInitialization(ProxyInitializeEvent event) {
 
 ## Audience operations not supported
 
-Velocity only supports sending chat messages, action bar messages, titles, and bossbars through the
-Adventure API. No other operations are supported.
+Velocity only supports sending chat messages, action bar messages, titles, bossbars, tablist header and footer, and resource packs through the
```
Comment (kashike) [inline docs/velocity/dev/getting-started/pitfalls.mdx:36]: Is it worth having a table here instead?

Comment (Timongcraft) [reply docs/velocity/dev/getting-started/pitfalls.mdx:36]: We could, but if Kiory adds something, it won't be listed and would need to be updated.

Comment (Timongcraft) [reply docs/velocity/dev/getting-started/pitfalls.mdx:36]: *Literal* or with a component?
I see both in the docs, so I'm not sure what I should use here.

**Review by zlataovce — APPROVED** [review]

**Comment by Leguan16** [comment]
The table alone after a heading looks weird. I think some sort of introduction to why some operations are not supported would be nice to have.

Due to <reason> and <reason> Velocity currently only supports the following audience operations:
<table>

**Comment by Timongcraft** [comment]
> The table alone after a heading looks weird. I think some sort of introduction to why some operations are not supported would be nice to have.
> 
> Due to and Velocity currently only supports the following audience operations:

You mean a general introduction before the table, right, and not a description for each operation?

**Comment by Leguan16** [comment]
> > The table alone after a heading looks weird. I think some sort of introduction to why some operations are not supported would be nice to have.
> 
> > 
> 
> > Due to and Velocity currently only supports the following audience operations:
> 
> 
> 
> You mean a general introduction before the table, right, and not a description for each operation?

Yes just a message before the table explaining why there are certain methods not implemented.

**Review by zlataovce — APPROVED** [review]

**Pull request merged**